src/Language/PureScript/Sugar/Names/Env.hs

```diff
@@ -60,16 +60,16 @@ data Imports = Imports
   , importedModules :: S.Set ModuleName
   -- |
   -- The names of "virtual" modules that come into existence when "import as"
-  -- is used.
+  -- is used, mapped to the real module being imported.
   --
-  , importedVirtualModules :: S.Set ModuleName
+  , importedVirtualModules :: M.Map ModuleName ModuleName
```

Comment:
That's totally fine by me, it shouldn't complicate the names stuff much, right?

We're already on the second revision of the names stuff and I don't think it's any clearer what is going on than my first attempt at it (although it works much better this time) - I'd like to try yet again to make it a little easier to comprehend and extract this kind of info out of, but that's not something I want to get into right now :wink: